dev_guide/builds/build_environment.adoc

```diff
@@ -37,14 +43,17 @@ env:
         fieldPath: metadata.name <3>
 ----
 
-<1> the value should be resolved using the following references
-<2> the following `fieldPath` refers to a jsonpath in the `build`
-<3> the jsonpath to the field in the `build`
+<1> `valueFrom` should be resolved using `fieldPath` and `fieldRef`.
+<2> `fieldPath` refers to a `jsonpath` in the `build`.
+<3> `jsonpath` to the field in the `build`.
+
+[NOTE]
+====
+'valueFrom' syntax does not yet support Jenkins Pipeline build environment variables.
```

Comment:
i'd reverse this. ("jenkins pipeline strategy does not support valueFrom syntax for environment variables" or "jenkins pipeline strategy env variables do not support valueFrom syntax"

(notice i removed the yet. i'd prefer to just say what is/isn't available.  if/when we add it, we'll change the doc, but there's no point in implying it's just around the corner)